tests/reporter_test.py

```diff
@@ -29,20 +29,33 @@ def setup_method(self, method):
 
         self.metrics_header = ["Metric", "Task", "Baseline", "Contender", "Diff", "Unit", "Diff %"]
         self.metrics_data = [
-            ["Min Throughput", "index", "17300", "18000", "700", "ops/s", "4.04%"],
-            ["Median Throughput", "index", "17500", "18500", "1000", "ops/s", "5.71%"],
-            ["Max Throughput", "index", "17700", "19000", "1300", "ops/s", "7.34%"],
+            ["Min Throughput", "index", 17300, 18000, 700, "ops/s", "4.04%"],
+            ["Median Throughput", "index", 17500, 18500, 1000, "ops/s", "5.71%"],
+            ["Max Throughput", "index", 17700, 19000, 1300, "ops/s", "7.34%"],
+            ["Bubblegum", "", 1.7, 1.9, 0.2, "bubbles/flavonoid", "11.71"],
```

Comment:
```suggestion
            ["Bubblegum", "", 1.7, 1.9, 0.2, "bubbles/flavonoid", "11.71%"],
```

(this will require changing the tests below)